src/main/java/com/myhome/helper/CommonHelper.java

```diff
@@ -0,0 +1,49 @@
+package com.myhome.helper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class CommonHelper {
+    public static String getStringFromObject(Object object) {
+        getObjectMapper().writer().withDefaultPrettyPrinter();
+        String jsonString = "";
+        try {
+            jsonString = getObjectMapper().writeValueAsString(object).replaceAll("\n", "");
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+
+        return jsonString;
+    }
+
+    public static ObjectMapper getObjectMapper() {
```

Comment:
Spring can autowire this for you. This can be removed and instead injected at the place this is going to be used.